django/db/migrations/writer.py

```diff
@@ -210,6 +211,11 @@ def as_string(self):
         # If there's a replaces, make a string for it
         if self.migration.replaces:
             items['replaces_str'] = "\n    replaces = %s\n" % self.serialize(self.migration.replaces)[0]
+        # Hinting that goes into comment
+        items.update(
+            version=get_version(),
+            timestamp=now().isoformat(sep=b' ' if six.PY2 else ' '),
```

Comment:
I don't know that timestamp would be that useful, but we keep it, I think the date might be sufficient -- at least seconds/milliseconds seems like too much information.